client/src/pages/admin/AddJudges.tsx

```diff
@@ -1,17 +1,24 @@
+import { useState } from 'react';
 import JuryHeader from '../../components/JuryHeader';
 import AddJudgeStatsPanel from '../../components/admin/add-judges/AddJudgeStatsPanel';
 import NewJudgeForm from '../../components/admin/add-judges/NewJudgeForm';
 import UploadCSVForm from '../../components/admin/add-judges/UploadCSVForm';
 
 const AddJudges = () => {
+    const [isJudgeAdded, setIsJudgeAdded] = useState<boolean>(false);
+
+    const handleAddJudge = () => {
+        setIsJudgeAdded(true);
+    }
+
     return (
         <>
             <JuryHeader withBack withLogout isAdmin />
             <div className="flex flex-col items-start justify-center w-full px-8 py-4 md:px-16 md:py-8">
                 <h1 className="text-4xl font-bold">Add Judges</h1>
-                <AddJudgeStatsPanel />
+                <AddJudgeStatsPanel key={isJudgeAdded.toString()} />
```

Comment:
What does this achieve? Adding a key does nothing to the component.

Reply:
it can redender the component when the judge is added

Reply:
This is such an unintuitive way of triggering a re-render in a component. Please use the global store.

Reply:
ok